action.yml

```diff
@@ -102,13 +103,13 @@ runs:
         retention-days: 5
 
     - name: Validate hyperlinks
+      if: ${{ fromJson(steps.prepare.outputs.validate).links }}
```

Comment:
Do we need to check something like `== true` or GitHub Actions understand this?

Reply:
Can you verify by running this on a fork of a repo (using `nschonni/spec-prod@VALIDATE_LINKS` instead of `w3c/spec-prod@v2`? (That's the best way to test GitHub Actions as far as I know, even though cumbersome).

Reply:
Do you have a particular repo to fork from? I'm not sure one that already has the right setup.
I'd probably suggest (separately), to create another workflow that just `uses: ./` to run as an integration test if there is a way to dry-run the action without actually publishing

Reply:
Let me do the fork thing :)
(For adding a new workflow, I'm lacking the time to invest right now, but will try to do sometime in future; thanks for the suggestion)

Reply:
You can see a rough idea in https://github.com/actions/setup-go/blob/main/.github/workflows/versions.yml

Reply:
Ok, I tested it in a fork. It works well 🎉 
(https://github.com/sidvishnoi/geolocation-api/pull/1)